packages/admin-ui/ui/src/domain/customers/details/index.tsx

```diff
@@ -36,87 +36,87 @@ const CustomerDetail = () => {
+            <div className="gap-x-base flex w-full items-center">
+              <div className="h-[64px] w-[64px]">
+                <Avatar
+                  user={customer}
+                  font="inter-2xlarge-semibold w-full h-full"
+                  color="bg-fuchsia-50"
+                />
+              </div>
+              <div className="flex grow flex-col">
+                <h1 className="inter-xlarge-semibold text-grey-90 max-w-[50%] truncate">
+                  {customerName()}
+                </h1>
+                <h3 className="inter-small-regular text-grey-50">
+                  {customer?.email}
+                </h3>
+              </div>
             </div>
-            <div>{moment(customer?.created_at).format("DD MMM YYYY")}</div>
+            <Actionables actions={actions} forceDropdown />
           </div>
-          <div className="flex flex-col pl-6">
-            <div className="inter-smaller-regular text-grey-50 mb-1">Phone</div>
-            <div className="max-w-[200px] truncate">
-              {customer?.phone || "N/A"}
+          <div className="mt-6 flex space-x-6 divide-x">
+            <div className="flex flex-col">
+              <div className="inter-smaller-regular text-grey-50 mb-1">
+                First seen
+              </div>
+              <div>{moment(customer?.created_at).format("DD MMM YYYY")}</div>
             </div>
-          </div>
-          <div className="flex flex-col pl-6">
-            <div className="inter-smaller-regular text-grey-50 mb-1">
-              Orders
+            <div className="flex flex-col pl-6">
+              <div className="inter-smaller-regular text-grey-50 mb-1">
+                Phone
+              </div>
+              <div className="max-w-[200px] truncate">
+                {customer?.phone || "N/A"}
+              </div>
             </div>
-            <div>{customer?.orders.length}</div>
-          </div>
-          <div className="h-100 flex flex-col pl-6">
-            <div className="inter-smaller-regular text-grey-50 mb-1">User</div>
-            <div className="h-50 flex items-center justify-center">
-              <StatusDot
-                variant={customer?.has_account ? "success" : "danger"}
-                title={customer?.has_account ? "True" : "False"}
-              />
+            <div className="flex flex-col pl-6">
+              <div className="inter-smaller-regular text-grey-50 mb-1">
+                Orders
+              </div>
+              <div>{customer?.orders.length}</div>
+            </div>
+            <div className="h-100 flex flex-col pl-6">
+              <div className="inter-smaller-regular text-grey-50 mb-1">
+                User
+              </div>
+              <div className="h-50 flex items-center justify-center">
+                <StatusDot
+                  variant={customer?.has_account ? "success" : "danger"}
+                  title={customer?.has_account ? "True" : "False"}
```

Comment:
question: Should we change the True/False? I never liked this, and I don't think it makes sense for a non-developer user 😅

Reply:
Would make sense, do you have any ideas for alternatives?

Reply:
Kill the dot and say "Registered customer" and nothing if `has_account = false` ?

Reply:
Sounds good!